nengo/spa/basalganglia.py

```diff
@@ -141,3 +143,36 @@ def add_dot_input(self, index, source, symbol, scale):
         with self.spa:
             nengo.Connection(output, self.input[index:index+1],
                              transform=transform, synapse=self.input_synapse)
+
+    def add_scalar_input(self, index, source):
+        """ Add a scalar input that will vary over time.
+
+        This would be used for such a thing as the ouput of the Compare module.
+
+        Parameters
+        ----------
+        index : int
+            the index of the action
+        source : Source
+            the module output to read from
+        symbol : Source
+            the semantic pointer to compute the dot product with
+        scale : float
+            a scaling factor to be applied to the result
+        """
+        output, _ = self.spa.get_module_output(source.name)
+        if(output.size_out != 1):
```

Comment:
Yeah. Parentheses and ```if``` statements don't work so well together. I do use them if the logic gets complex. Also, you can't use them to split lines because:
```
if (cond1 or
    cond2):
    do_things = True
```
Will throw a flake8 error (all the tabs are aligned).